### ISSUE TITLE
Travis not being triggered.

### ISSUE THREAD

**Issue by rbong** [coderanger/nightlies#13]
I have logged into nightli.es with my github account, given it all permissions, and turned it on for my desired repository.

<img width="974" alt="screen shot 2016-04-07 at 8 40 07 pm" src="https://cloud.githubusercontent.com/assets/5875706/14371095/f3432de2-fd01-11e5-8050-d59f914d5d1f.png">

I have been waiting over a day and it has not yet built.

**Comment by LuisAlejandro** [comment]
Same here.

**Comment by linuxenko** [comment]
Sad, but it seems like night**_lies**_ does not work

**Comment by coderanger** [comment]
Please include the repo name if you're having issues.

**Comment by coderanger** [comment]
@rbong Didn't notice this ticket at the time, sorry. I do see your repo in the logs though:

```
Oct 15 13:06:19 nightlies app/scheduler.4478: [rbong/livestreamer-nightly] Requesting a build, last build time 2016-10-14 20:06:05 +0000. 
```

Are you still not seeing API builds?

**Comment by rbong** [comment]
No- in fact I can't access livestreamer-nightly on the nightli.es dashboard anymore. Travis still indicates that the last build was 9 months ago.
EDIT: Apologies; the repo was disabled in Travis. I'll see if it builds now.

**Comment by rbong** [comment]
Unfortunately it seems the repository still did not build.

**Comment by coderanger** [comment]
@rbong It looks like you don't have a branch called master. Currently this is hard coded as the branch to be built. There is a separate ticket (#1 in fact) about fixing that.
